docs/v3/tutorials/server-and-worker.mdx

```diff
@@ -128,14 +124,13 @@ worker:
   config:
     workPool: kube-test
   selfHostedServerApiConfig:
-    # SERVICE.NAMESPACE if in same cluster, or FQDN if external
-    apiUrl: http://prefect-server.prefect-server.svc.cluster.local:4200/api
-  basicAuth:
-    enabled: true
-    existingSecret: worker-auth-secret
+    apiUrl: http://prefect-server.prefect.svc.cluster.local:4200/api
+    basicAuth:
```

Comment:
this nesting was just introduced in our latest release, adjusting here for accuracy